cvat/apps/engine/permissions.py

```diff
@@ -196,15 +200,17 @@ def get_resource(self):
         elif self.obj:
             data = {
                 'id': self.obj.id,
-                'owner': { 'id': getattr(self.obj.owner, 'id', None) },
+                'owner': { 'id': self.obj.owner_id },
                 'organization': {
-                    'id': self.obj.organization.id
-                } if self.obj.organization else None
+                    'id': self.obj.organization_id
+                } if self.obj.organization_id else None
```

Comment:
I would suggest to add a common function for this pattern, since it seems to be repeating a lot:

```python
def id_dict(id):
    if id is None: return None
    return {'id': id}
```